### PR TITLE
Add logo for new sources

### LINKED ISSUE #1015: Add logos for new Sources
## Problem

We have the following sources come online soon:
- NASA (nasa.gov)
- Biodiversity Library (https://www.biodiversitylibrary.org/)
- SMK (https://www.smk.dk/)
- Science Museum (https://www.sciencemuseumgroup.org.uk/)
-- _Note this may already exist, since we previously integrated, but updated the integration now_
- Europeana (https://www.europeana.eu/en)

We need to make sure that we have the logos set up in the front end for them.

Note also, that currently the Collections page is split into "Museums" and "Other Collections".

NASA and Biodiversity Library should be in the "Other Collections" section, and SMK, Science Museum, and Europeana should be in the Museums section.

### PR DESCRIPTION
**Pull request by Tanuj22**
## Fixes

Fixes #1015 #1021 by @mathemancer 

## Description
Add Logos for new sources

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by Tanuj22** [comment]
Science Museum already existed

**Comment by kgodey** [comment]
@Tanuj22 can you post a screenshot?

File: src/api/ImageProviderService.js [hunk 1]
```diff
@@ -86,6 +86,21 @@ const ImageProviderService = {
       smithsonian: {
         logo: 'smithsonian_logo.svg',
       },
+      bio_diversity: {
```
Comment (kgodey) [inline src/api/ImageProviderService.js:89]: please keep providers in alphabetical order

**Comment by kgodey** [comment]
@Tanuj22 thanks for your work on this! I'll make the fixes here so that we can do a release today.

**Pull request merged**